iptables fails to start

**Describe the bug**
Getting the following in the logs:
```
Apr 03 14:34:58 test-fresh.inverse packetfence[21725]: WARN -e(21725): Problem trying to run command: LANG=C /sbin/iptables-restore < /usr/local/pf/var/conf/iptables.conf called from iptables_restore. Child exited with non-zero value 1 (pf::util::pf_run)
```

```
[root@localhost pf]# LANG=C /sbin/iptables-restore < /usr/local/pf/var/conf/iptables.conf
iptables-restore: line 205 failed
```

Line 205 of /usr/local/pf/var/conf/iptables.conf:
```
COMMIT
```

My full iptables.conf: https://support.inverse.ca/~jsemaan/iptables.conf

On Debian, it seems necessary to run:
```
apt install linux-headers-$(uname -r)
```

I'm running latest version of kernel headers for Debian (`4.9.210-1`) with `iptables-netflow-dkms` package installed from packetfence repo and still have the issue.

During PacketFence installation on Debian, I noticed following error message:
```
Setting up iptables-netflow-dkms (2.4-1) ...                                                                                                                                                                      
Loading new ipt-netflow-2.4 DKMS files...                                                                                                                                                                         
Building for 4.9.0-12-amd64                                                                                                                                                                                       
Building initial module for 4.9.0-12-amd64                                                                                                                                                                        
Error! Bad return status for module build on kernel: 4.9.0-12-amd64 (x86_64)                                                                                                                                      
Consult /var/lib/dkms/ipt-netflow/2.4/build/make.log for more information. 
```
but it doesn't stop installation.

Log file:
```
DKMS make.log for ipt-netflow-2.4 for kernel 4.9.0-12-amd64 (x86_64)
Tue Apr  7 06:03:12 UTC 2020
make: *** No rule to make target 'gen_compat_def', needed by 'compat_def.h'.  Stop.
```

--- a/UPGRADE.asciidoc
+++ b/UPGRADE.asciidoc
@@ -1090,26 +1090,27 @@ cat /usr/local/pf/conf/pf-release > /usr/local/pf/conf/currently-at
 
 == Upgrading from a version prior to X.Y.Z
 
-=== Kernel-devel ===
+=== Kernel development package
 
-In this version we need to have the kernel-devel package that match your current kernel version in order to build the netflow kernel module.
+NOTE: This step need to be done *before* packages upgrade.
 
-Before doing the upgrade you have 2 choices:
+In this version we need to have the kernel development package that match your current kernel version in order to build the Netflow kernel module.
 
-[source,bash]
-----
-yum update
-----
-and reboot the server
-
-or
+==== RHEL / CentOS based systems
 
 [source,bash]
 ----
-yum install "kernel-devel-uname-r == $(uname -r)"
+yum install kernel-devel-$(uname -r)
 ----
 
-=== Timezone ===
+==== Debian based systems
+
+[source,bash]
+----
+apt install linux-headers-$(uname -r)
+----
+
+=== Timezone
 
 The timezone set in pf.conf will be set on the operating system every time PacketFence reloads its configuration.
 For this reason, you should review the timezone setting in the general section of pf.conf (System Configuration -> General Configuration in the admin).

--- a/UPGRADE.asciidoc
+++ b/UPGRADE.asciidoc
@@ -990,7 +990,7 @@ Once completed, update the file /usr/local/pf/conf/currently-at to match the new
 
 === Merge of all RPM packages into one (RHEL / CentOS only)
 
-NOTE: This step need to be done *before* packages upgrade.
+NOTE: This step needs to be done *before* packages upgrade.
 
 Starting from now, PacketFence will be released as an unique RPM package for
 `x86_64` architectures. To remove properly older RPM packages, you need to follow these steps:
@@ -1025,7 +1025,7 @@ installed, this one has been upgraded to latest version.
 
 === New GPG key for Debian installations (Debian only)
 
-NOTE: This step need to be done *before* packages upgrade.
+NOTE: This step needs to be done *before* packages upgrade.
 
 In order to install new versions of Debian packages, you will need to add a new GPG key to your system: 
 
@@ -1092,7 +1092,7 @@ cat /usr/local/pf/conf/pf-release > /usr/local/pf/conf/currently-at
 
 === Kernel development package
 
-NOTE: This step need to be done *before* packages upgrade.
+NOTE: This step needs to be done *before* packages upgrade.
 
 In this version we need to have the kernel development package that match your current kernel version in order to build the Netflow kernel module.
 

--- a/docs/PacketFence_Installation_Guide.asciidoc
+++ b/docs/PacketFence_Installation_Guide.asciidoc
@@ -183,22 +183,6 @@ Red Hat-based Systems
 
 NOTE: Applies to CentOS and Scientific Linux but only the x86_64 architecture is supported.
 
-Explicitly instruct NetworkManager to never interact with your DNS configuration (https://bugzilla.redhat.com/show_bug.cgi?id=1344303[Source]):
-
-[source,bash]
-----
-echo "[main]
-dns=none" > /etc/NetworkManager/conf.d/99-no-dns.conf
-service NetworkManager restart
-----
-
-You also need to disable systemd-logind and reboot:
-
-[source,bash]
-----
-/usr/bin/systemctl --now mask systemd-logind
-reboot
-----
 
 RHEL 7.x
 ++++++++

--- a/docs/PacketFence_Installation_Guide.asciidoc
+++ b/docs/PacketFence_Installation_Guide.asciidoc
@@ -130,8 +130,7 @@ Import to VMware Player/Workstation for Linux
 Newer version of VMware Player handles the VLAN trunking a lot better.  Having that said, we can use a single interface on the VM.  So, you need to ensure that your VM host is plugged into a physical trunk port with VLAN 1,2,3,5,10 and 200 as the allowed VLAN. These VLANs will be used later in configuration examples.
 
 
-Installing PacketFence on existing Linux
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+=== Installing PacketFence on existing Linux
 
 PacketFence provides packages repository for RHEL / CentOS as well as packages repository fo Debian.
 
@@ -178,14 +177,18 @@ link:https://wiki.debian.org/AppArmor/HowToUse#Disable_AppArmor[Debian wiki].
 
 Regarding resolvconf, you can remove the symlink to that file and simply create the `/etc/resolv.conf` file with the content you want.
 
-Red Hat-based Systems
-^^^^^^^^^^^^^^^^^^^^^
+==== RHEL / CentOS based systems
 
 NOTE: Applies to CentOS and Scientific Linux but only the x86_64 architecture is supported.
 
+Install kernel development package:
 
-RHEL 7.x
-++++++++
+[source,bash]
+----
+yum install kernel-devel-$(uname -r)
+----
+
+===== RHEL 7.x
 
 NOTE: These are extra steps are required for RHEL 7 systems only, excluding derivatives such as CentOS or Scientific Linux.
 
@@ -197,11 +200,18 @@ subscription-manager repos --enable rhel-7-server-optional-rpms
 subscription-manager repos --enable rhel-7-server-extras-rpms
 ----
 
-Software Installation
-^^^^^^^^^^^^^^^^^^^^^
+==== Debian based systems
 
-RHEL / CentOS
-+++++++++++++
+Install kernel development package:
+
+[source,bash]
+----
+apt install linux-headers-$(uname -r)
+----
+
+==== Software Installation
+
+===== RHEL / CentOS based systems
 
 In order to use the PacketFence repository:
 
@@ -219,8 +229,7 @@ server, DHCP server, RADIUS server) using:
 yum install --enablerepo=packetfence packetfence
 ----
 
-Debian
-++++++
+===== Debian based systems
 
 In order to use the repository, create a file named [filename]`/etc/apt/sources.list.d/packetfence.list`:
 

